fastapi_pagination/api.py

```diff
@@ -361,7 +363,11 @@ def _update_route(route: APIRoute) -> None:
         ),
     )
 
-    route.body_field = get_body_field(dependant=route.dependant, name=route.unique_id)
+    if fastapi_version >= '0.112.4':
```

Comment:
`importlib.metadata.version` returns a string. str1 > str2 is not correct way to compare versions.
For example:
```python
'3.9.5' > '3.10.3'
Out[1]: True

Version('3.9.5') > Version('3.10.3')
Out[59]: False
```

We need to use `packaging.version.Version` instead:
```python
from packaging.version import Version

v1 = Version('0.112.2')
v2 = Version('0.112.4')

assert v2 > v1
```

Reply:
My bad, you are right! I have implemented the version check using packaging.version :)